IMC_WebDev/#Driving_the_car_and_flowing_the_water_with_CSS/blog.md

```diff
@@ -0,0 +1,136 @@
+# Introduction to Web Animations using CSS
+
+In this blog, we are going to keep building our mini city. We will have a car going along the road and the bridge, and a boat going under the bridge.
+
+![](https://i.imgur.com/9FFPgkl.gif)
+
+
+
+## Build the car and the boat
+
+First, let's build our car and boat with HTML and CSS like we did for the ground and buildings previously.
+The car is a rectangle box and the boat is a rectangle box with a triangle in the front. Both of them has a shadow to help show the 3D effect.
+So we need to have 2 divs for the car and the boat, and in each of them, we will have some children divs inside.
```

Comment:
Briefly mention why there are children divs. What are you going to do with them?